execution/gethexec/sequencer.go

```diff
@@ -196,6 +198,7 @@ var DefaultDangerousConfig = DangerousConfig{
 
 func SequencerConfigAddOptions(prefix string, f *flag.FlagSet) {
 	f.Bool(prefix+".enable", DefaultSequencerConfig.Enable, "act and post to l1 as sequencer")
+	f.Duration(prefix+".read-from-tx-queue-timeout", DefaultSequencerConfig.ReadFromTxQueueTimeout, "time duration after which the sequencer stops reading txs from queue")
```

Comment:
sequencer doesn't really wait this long. More correct to say something likr "timeout for reading new messages". It's o.k. that it's not very clear because it's complicated to explain and people should generally not touch it.